src/assets/data/fault-summary-lookup.json

```diff
@@ -47,5 +47,13 @@
   "trafficControllers": "Traffic controllers",
   "trafficLights": "Traffic lights",
   "trafficSigns": "Traffic signs",  
-  "vehicleCheck": "Vehicle check"
+  "vehicleCheck": "Vehicle check",
+  "manoeuvreRRCtrl": "Manoeuvre - Reverse Right - Control",
+  "manoeuvreRRObs": "Manoeuvre - Reverse Right - Observation",
+  "manoeuvreRPRCtrl": "Manoeuvre - Reverse Park (Road) - Control",
+  "manoeuvreRPRObs": "Manoeuvre - Reverse Park (Road) - Observation",
+  "manoeuvreRPCCtrl": "Manoeuvre - Reverse Park (Car park) - Control",
+  "manoeuvreRPCObs": "Manoeuvre - Reverse Park (Car park) - Observation",
+  "manoeuvreFPCtrl": "Manoeuvre - Forward Park - Control",
+  "manoeuvreFPObs": "Manoeuvre - Forward Park - Observation"
```

Comment:
I fixed this in develop too - I prefer your ordering so overwrite it :)

Reply:
done :)